cypress/e2e/home.cy.ts

```diff
@@ -1,8 +1,67 @@
-const projectUrl = Cypress.env("BASE_URL");
+import { projectUrl, stellarIconUrl } from "../support/constants";
 
-describe("Navigation", () => {
-  it("Go to the website:", () => {
-    console.log(projectUrl, "ProjectURL");
-    cy.visit(projectUrl);
+beforeEach(() => {
+  cy.visit(projectUrl);
+});
+
+describe("Home: UI and functionality:", () => {
+  it("Home UI: Stellar logo", () => {
+    cy.get('[data-cy="stellar-home-logo"]').should("be.visible");
+  });
```

Comment:
Add an enter here, please.

Reply:
issue solved in commit f0a74323da3f19ea9b248e1159579dd2625abb9d.